internal/pkg/utils/utils.go

```diff
@@ -60,8 +62,15 @@ func ValidateSTACKITURL(value string) error {
 	if urlHost == "" {
 		return fmt.Errorf("bad url")
 	}
-	if !strings.HasSuffix(urlHost, "stackit.cloud") {
-		return fmt.Errorf(`only urls belonging to STACKIT are allowed, hostname must end in "stackit.cloud"`)
+
+	allowedUrlDomain := viper.GetString(config.AllowedUrlDomainKey)
+
+	if allowedUrlDomain == "" {
+		allowedUrlDomain = config.AllowedUrlDomainDefault
+	}
```

Comment:
Now that a default is available this is probably not needed, unless we really want to disallow users to "allow any domain" by setting the flag to `""`

Reply:
I think it should not be allowed to set it to empty.

Reply:
In this case the user is setting explicitly the allowed domain to be empty, so yeah its like an "allow any domain" configuration. 
I think it could be misleading to just set the default in this case but I agree that it is a security risk. I would suggest a middle term:
- add a warning when the user sets this config option to empty, but not use the default here

Reply:
removed condition and added warning